.github/workflows/deploy.yml

```diff
@@ -0,0 +1,60 @@
+name: Deploy
+
+on:
+  workflow_run:
+    workflows: ["CI"]
+    types:
+      - completed
+
+jobs:
+  upload:
+    name: Push template to master
+    runs-on: ubuntu-latest
+    if: github.repository == 'astropy/package-template' && github.event.workflow_run.event == 'push' && github.event.workflow_run.conclusion == 'success'
+    steps:
+    - name: Set up Git
+      run: |
+        git --version
+        git config --global user.name "github-actions[bot]"
+        git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+    - name: Download artifact
+      uses: actions/github-script@v3
+      with:
+        script: |
```

Comment:
There's js in my yaml :scream_cat:

Reply:
Yes, unfortunate due to actions/download-artifact#60

Reply:
Linking that issue from the script might be a good idea, so whichever poor soul ends up having to edit this in the future could see if it's fixed upstream?

Reply:
Done. What about now?